modules/installation-dns-user-infra.adoc

```diff
@@ -158,3 +160,8 @@ $TTL 1W
 ;
 ;EOF
 ====
+
+[NOTE]
+====
+For IPI installations, only the DNS records need to be added.
```

Comment:
We avoid using `IPI` and instead spell out the acronym: installer-provisioned infrastructure. So perhaps something like this instead?:
```
For clusters using installer-provisioned infrastructure, only the DNS records must be added.
```